lib/hotel_system.rb

```diff
@@ -0,0 +1,133 @@
+require 'date'
+
+require_relative 'room'
+require_relative 'reservation'
+require_relative 'block'
+
+module Hotel
+  class HotelSystem
+    attr_reader :rooms, :reservations, :blocks
+    
+    def initialize
+      @rooms = []
+      @reservations = []
+      @blocks = []
+      
+      20.times do |i|
+        @rooms << Hotel::Room.new((i + 1))
```

Comment:
Ruby will be looking for names locally first, prioritizing local names, since they're all in the same module, there's no need to namespace `Hotel::Room`. `Room` should just work! Same with the rest of the namespacing in the code.